boards/arm/nucleo_h745zi_q/nucleo_h745zi_q_m4.yaml

```diff
@@ -0,0 +1,18 @@
+identifier: nucleo_h745zi_q_m4
+name: NUCLEO-H745ZI-Q
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 288
+flash: 1024
+supported:
+  - arduino_gpio
+  - gpio
+testing:
+  ignore_tags:
+    - mpu
+    - nfc
+    - net
```

Comment:
Is that required ?
Do you have failing tests reported if not setting these?

Reply:
I just copy-pasted it from h747 disco. Since I am new in Zephyr could not find exact testing description on CI. Can you send me a link please?

Reply:
deleted ignored tags

Reply:
I can't see ignored tags deleted. Are they required?

Reply:
> I can't see ignored tags deleted. Are they required?

The following `ignore_tags` were mentioned:

`ignore_tags:`
`     - mpu`
`     - nfc`
`     - net`

Yes, they are required, since other way the corresponding tests fail. Same approach used by `stm32h747_disco`, as I mentioned it before.